eventlet/wsgi.py

```diff
@@ -967,6 +977,7 @@ def server(sock, site,
         url_length_limit=url_length_limit,
         debug=debug,
         socket_timeout=socket_timeout,
+        idle_timeout=idle_timeout,
```

Comment:
Personally I like "idle" term better, but it's widely recognized as "keepalive".

idle_timeout -> keepalive_timeout ?

and then we could gradually surrender the `keepalive=False` option to `..._timeout=0`

Reply:
Sure -- I think that opt had me thinking of `SO_KEEPALIVE`, and I forgot that I already [take care of that](https://github.com/openstack/swift/blob/2.31.0/swift/common/wsgi.py#L215) before calling `server` 😃

Makes me realize I might consider sending a [`Keep-Alive` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive), and maybe also adding a `keepalive_max` opt.

Reply:
Renamed, and started the deprecation process. Also went ahead and added the `Keep-Alive` header, provided that we were going to send the `Connection` header anyway.

Reply:
Seems convenient, can't think of issues with double typed bool/float `keepalive`.